projects/go-lib/src/lib/go-shared.module.ts

```diff
@@ -8,13 +8,15 @@ import { GoModalModule } from './components/go-modal/go-modal.module';
 import { GoTableModule } from './components/go-table/go-table.module';
 import { GoToastModule } from './components/go-toast/go-toast.module';
 import { GoToasterModule } from './components/go-toaster/go-toaster.module';
+import { GoIconButtonModule } from './components/go-icon-button/go-icon-button.module';
 
 @NgModule({
   imports: [
     GoAccordionModule,
     GoButtonModule,
     GoCardModule,
     GoIconModule,
+    GoIconButtonModule,
```

Comment:
For the sake of consistency, could we alphabetize this file? This spot, the import above, and the exports below.